boomer/src/main.rs

```diff
@@ -228,6 +230,12 @@ fn main() -> Result<()> {
     .context("Couldn't create our framebuffers")?;
 
     info!("Setting up the pipeline");
+    // debug!("Output: {:?}", output);
+    // debug!("Connector: {:?}", connector);
+    // debug!("Mode: {:?}", mode);
+    // debug!("Plane: {:?}", plane);
+    // debug!("Width: {}", width);
+    // debug!("Height: {}", height);
```

Comment:
Why do we need these changes?

Reply:
I `git commit --amend`ed them but forgot to force push :man_facepalming: